docs/_static/style.css

```diff
@@ -1,4 +1,4 @@
-@import url("theme.css");
+@import url("styles/furo.css");
```

Comment:
slight theme regression, don't know if it's important: the "external link" css is gone
in the current docs:
![Screenshot_20240429_112025](https://github.com/Ouranosinc/xclim/assets/91751417/9b80bc56-50fb-49cb-9d4e-639bcb0771c7)
in furo:
![Screenshot_20240429_112128](https://github.com/Ouranosinc/xclim/assets/91751417/c5aac263-aabf-44ab-adfd-728b0ceb008a)

Reply:
The styling of external links was an option specifically for ReadTheDocs theme. Will see what I can do about this.

Reply:
It appears that there's a discussion for this: https://github.com/pradyunsg/furo/discussions/775